src/vma/sock/sock-redirect.cpp

```diff
@@ -2107,6 +2107,12 @@ pid_t fork(void)
 
 		safe_mce_sys().get_env_params();
 		vlog_start("VMA", safe_mce_sys().log_level, safe_mce_sys().log_filename, safe_mce_sys().log_details, safe_mce_sys().log_colors);
+#ifdef RDMA_LIB_RESET
+		if (rdma_lib_reset()) {
+			srdr_logerr("Child Process: rdma_lib_reset failed %m",
+					errmo);
```

Comment:
errmo ???
If RDMA_LIB_RESET was defined you would have failed on compilation.
sock/sock-redirect.cpp:2112: error: 'errmo' was not declared in this scope

Please explain how you tested fork() with rdma_lib_reset()